extensions/amp-3q-player/0.1/amp-3q-player.js

```diff
@@ -86,17 +83,42 @@ class Amp3QPlayer extends AMP.BaseElement {
     Services.videoManagerForDoc(el).register(this);
   }
 
-  /** @override */
-  layoutCallback() {
-    const iframe = createFrameFor(
-      this,
-      'https://playout.3qsdn.com/' +
+  /** private */
```

Comment:
This should be `/** @private */`, and the method should be suffixed with an underscore: `generateIframeSrc_()`